src/engine/controls/cuecontrol.cpp

```diff
@@ -854,13 +856,13 @@ void CueControl::hotcueSet(HotcueControl* pControl, double value, HotcueSetMode
         setCurrentSavedLoopControlAndActivate(pControl);
     }
 
-    // If quantize is enabled and we are not playing, jump to the cue point
+    // If snap is enabled and we are not playing, jump to the cue point
     // since it's not necessarily where we currently are. TODO(XXX) is this
     // potentially invalid for vinyl control?
     bool playing = m_pPlay->toBool();
-    if (!playing && m_pQuantizeEnabled->toBool()) {
+    if (!playing && m_pSnapEnabled->toBool()) {
         lock.unlock(); // prevent deadlock.
-        // Enginebuffer will quantize more exactly than we can.
+        // Enginebuffer will snap more exactly than we can.
         seekAbs(cueStartPosition);
     }
```

Comment:
Are you sure this block should be changed to snap? This seeks the deck which I think should solely be done by quantize.